Changelog.md

```diff
@@ -2,6 +2,35 @@
 
 ### libsdformat 9.X.X (202X-XX-XX)
 
+### libsdformat 9.6.1 (2021-09-07)
+
+1. Parse URDF continuous joint effort/velocity limits
+    * [Pull request #684](https://github.com/ignitionrobotics/sdformat/pull/684)
+
+1. Add a codecheck make target
+    * [Pull request #682](https://github.com/ignitionrobotics/sdformat/pull/682)
+
+1. Refactor sdf::readXml
+    * [Pull request #681](https://github.com/ignitionrobotics/sdformat/pull/681)
+
+1. Upgrade cpplint and fix new errors
+    * [Pull request #680](https://github.com/ignitionrobotics/sdformat/pull/680)
+
+1. BUG: add missing plugin element to include
+    * [Pull request #675](https://github.com/ignitionrobotics/sdformat/pull/675)
+
+1. Update codecov badge url
+    * [Pull request #678](https://github.com/ignitionrobotics/sdformat/pull/678)
+
+1. Added comment reminder to update functions
+    * [Pull request #677](https://github.com/ignitionrobotics/sdformat/pull/677)
+
+1. Port to new codecov configuration
```

Comment:
nitpick: my personal feeling is that we can save users from infra or other internal changes not relevant to the software delivered.

Reply:
I've redacted a few entries in https://github.com/ignitionrobotics/sdformat/pull/691/commits/32ce16a81e707a5000a13cbec686371689b336f7